doc/manuals/aggregated-data-retrieval.md

```diff
@@ -19,9 +19,11 @@ The requests for aggregated time series context information can use the followin
 
 -   **aggrMethod**: The aggregation method. The STH component supports the following aggregation methods: `max` (maximum
     value), `min` (minimum value), `sum` (sum of all the samples) and `sum2` (sum of the square value of all the
-    samples) for numeric attribute values and `occur` for attributes values of type string. Combining the information
-    provided by these aggregated methods with the number of samples, it is possible to calculate probabilistic values
-    such as the average value, the variance as well as the standard deviation. It is a mandatory parameter.
+    samples) for numeric attribute values and `occur` for attributes values of type string. It accepts multiple values
+    separated by comma (min,max) to get multiple aggregation method values. Additionally, `aggrMethod=all` can be used
```

Comment:
```suggestion
    separated by comma (eg. `aggrMethod=min,max`) to get multiple aggregation method values. Additionally, `aggrMethod=all` can be used
```

Reply:
Fixed in [3b51b4c](https://github.com/telefonicaid/fiware-sth-comet/pull/605/commits/3b51b4c42e14b77f331525db1fcd0926a79afffd)